odd-platform-ui/src/components/shared/EntityClassItem/EntityClassItemStyles.ts

```diff
@@ -1,9 +1,9 @@
 import styled from 'styled-components';
-import { DataEntityTypeNameEnum } from 'generated-sources';
+import { DataEntityClassNameEnum } from 'generated-sources';
 import { Box } from '@mui/material';
 
 export const Content = styled(Box)<{
-  typeName: DataEntityTypeNameEnum;
+  typeName: DataEntityClassNameEnum;
```

Comment:
```suggestion
  entityClassName: DataEntityClassNameEnum;
```

and add $ prefix for styled-component props, please

Reply:
done